flink-runtime/src/test/java/org/apache/flink/runtime/testutils/MiniClusterResource.java

```diff
@@ -186,6 +190,12 @@ private void startMiniCluster() throws Exception {
         configuration.setInteger(JobManagerOptions.PORT, 0);
         configuration.setString(RestOptions.BIND_PORT, "0");
 
+        // randomize ITTests for enabling buffer de-bloating
+        if (RANDOMIZE_BUFFER_DEBLOAT_CONFIG
```

Comment:
It's a pity we need to randomize tests in two different places. Can we at least cross reference the two in a comment? We should also try to explain which place we should use for what class of options.

Reply:
Yes, it is indeed a pity but since we can not configure the flink from one place so we can not randomize it from one place. 
I will see what I can do for the explanation of the usages.